src/sections/about.html

```diff
@@ -0,0 +1,19 @@
+<div id="about" class="mx-8 sm:mx-28 lg:mx-52 mt-28 lg:mt-72">
+  <p class="text-2xl md:text-3xl lg:text-5xl">
+    We are a Berlin-based communications agency specialising in helping
+    companies and brands find their voice and position themselves. Whether it's
+    brand building, identity sharpening, or communication strategy, we bring a
+    fresh perspective to established structures, develop tailor-made
+    communication solutions, and ensure that messages resonate. With over a
+    decade of experience, we have honed our skills in giving brands their
+    unmistakable identity and making them visible in the long term.
+  </p>
+</div>
+
+<div class="mx-8 sm:mx-28 lg:mx-52 sticky top-0">
+  <img
+  class="w-2/6 md:w-1/4 mx-auto mt-[86px]"
```

Comment:
Keep in mind that it's good to reserve the use of arbitrary values when you can default tailwind utility classes that has pretty close value.

```suggestion
  <img
  class="w-2/6 md:w-1/4 mx-auto mt-20"
```